views/home/references.ejs

```diff
@@ -363,7 +366,7 @@
           <td>
               <input type="checkbox" ng-change="approve(reference.id, reference.approved)"
                      ng-model="reference.approved" ng-if="user.isMod"/>
-              <span ng-if="user.isMod && refUser && reference.edited">*</span>
+              <span ng-if="user.isMod && refUser && reference.edited"><font color="red" title="This reference was edited at some point.">*</font></span>
```

Comment:
This PR has reminded me how much this file needs to be tidied up. There. Is. So. Much. Duplicated. Code. And it makes me sad.